cmd/infra/powervs/destroy.go

```diff
@@ -218,6 +213,11 @@ func (options *DestroyInfraOptions) DestroyInfra(ctx context.Context, infra *Inf
 		}
 	}
 
+	if err = deleteSecrets(options.Name, options.Namespace, powerVsCloudInstanceID, accountID, resourceGroupID); err != nil {
```

Comment:
Any reason to move this after deleteCOS?

Reply:
Yes, deleteSecrets needs `powerVsCloudInstanceID` value, which it is getting from the code after deleteCOS.